### DISCUSSION TITLE
Introduce New features

### DISCUSSION THREAD

**Discussion by ishwar-chandra** [community/community#15502] [Mobile]
Implement new features for code editing from our repository in GitHub app.

And introduce the cloud coding environment through GitHub app

I hope you do something about this suggestion

Thank you GitHub

I like GitHub service.

**Comment by queenofcorgis** [comment]
Hi there - what features would you like to see for code editing?

**Reply by ishwar-chandra** [reply]
I want code editor, and git commit, git push options in this app

**Reply by ishwar-chandra** [reply]
Also we want to create new repo in this app

**Reply by eliperkins** [reply]
GitHub for iOS now includes code editing within repositories: https://github.blog/changelog/2022-11-15-github-for-ios-edit-files-from-browse-code/

The feature will be coming soon on GitHub for Android as well!
